src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java

```diff
@@ -143,6 +147,7 @@ else if (flaggedRelations.size() != 1 && !feature.has(GEOMETRY))
         flagProperties.add("feature_properties", featureProperties);
         flagProperties.add("feature_osmids", uniqueFeatureOsmIds);
         flagProperties.addProperty("feature_count", featureProperties.size());
+        flagProperties.add(IDENTIFIERS, GSON.toJsonTree(flag.getUniqueIdentifiers()));
```

Comment:
Yes it is converting it to a json array. 
I think there are a few advantages. It makes it easier to read and prettify when in json format. There is no issue with trying to deal with gigantic unbroken strings.  A lot of languages have no problem loading a json array into a native collection format, so they can then easily use the ids in a separated (search) or combined (id) manor. 
That said, it would probably work just about as well to have his as a csv string. For an example of how it is currently used by downstream tools look at the changes to the geojson and log diff tools in this PR.